whylog/config/__init__.py

```diff
@@ -28,6 +35,39 @@ def _filter_rule_set(self, parsers_list):
     def _get_locations_for_logs(self, logs_types_list):
         pass
 
+    def _load_parsers(self):
+        parsers_definitions = self._load_file_with_config(self._parsers_path)
+        log_type_manager = LogTypeManager()
```

Comment:
Initialize in constructor and reference through `self._log_type_manager` rather than initializing it here and passing it down. LogTypeManager can register new types and eventually be asked about existing types, so that configuration can be updated (to add new types).